pkgs/tools/package-management/nix-folder2channel/default.nix

```
@@ -0,0 +1,34 @@
+{
+  stdenv, writeScriptBin, buildEnv,
+  git,
+}:
+let
+  nix-folder2channel-script = writeScriptBin "nix-folder2channel" ''
+    #!${stdenv.shell}
+    if ! [ -f default.nix ]; then
+      echo Please run at root of nixpkgs
+      exit 1
+    fi
+
+    HASH=`nix-build nixos/release.nix -A channel --arg nixpkgs  '{ outPath = ./. ; revCount = "'$(git rev-list HEAD | wc -l)'"; shortRev = "'$(git rev-parse --short HEAD)'"; }' | awk -F'/' '{print $4}'`
```

```suggestion
    HASH=$(nix-build nixos/release.nix -A channel --arg nixpkgs  '{ outPath = ./. ; revCount = "'$(git rev-list HEAD | wc -l)'"; shortRev = "'$(git rev-parse --short HEAD)'"; }' | awk -F'/' '{print $4}')
```

Or is this not available?

---

Thanks for review.